dropwizard-client/src/main/java/io/dropwizard/client/JerseyClientConfiguration.java

```diff
@@ -12,7 +12,7 @@
  * {@link HttpClientConfiguration}.
  *
  * @see HttpClientConfiguration
- * @see <a href="http://dropwizard.io/0.9.1/docs/manual/configuration.html#jerseyclient">Jersey Client Configuration</a>
+ * @see <a href="http://dropwizard.io/1.0.1/docs/manual/configuration.html#jerseyclient">Jersey Client Configuration</a>
```

Comment:
It'd be nice to link this to a place that's not version specific, but if we are going to update it, would you mind updating it to the latest release `1.0.2`?